crates/trippy-tui/src/report/table.rs

```diff
@@ -42,20 +51,25 @@ fn run_report_table<R: Resolver>(
         .set_header(columns);
     for hop in trace.hops() {
         let ttl = hop.ttl().to_string();
-        let ips = hop.addrs().join("\n");
-        let ip = if ips.is_empty() {
-            String::from("???")
+        let (ip, host) = if privacy_max_ttl > 0 && hop.ttl() <= privacy_max_ttl {
```

Comment:
@ObsidianAli the check should be `privacy_max_ttl >= Some(hop.ttl())` where `privacy_max_ttl` is passed in as `Option<u8>`.